src/ssl.c

```diff
@@ -7899,6 +7922,7 @@ int wolfSSL_DTLS_SetCookieSecret(WOLFSSL* ssl,
             }
 #endif /* NO_HANDSHAKE_DONE_CB */
 
+#ifndef WOLFSSL_WPAS
```

Comment:
Is the build time switch WOLFSSL_WPAS the right way to determine whether to free handshake resources or not?  What about a user doing TLS with this build but not in WPAS mode?